ansible/roles/openfisca_api_fr/templates/cron.d/openfisca-api-fr.j2

```diff
@@ -0,0 +1,3 @@
+# Every day, run the Ansible playbook that installs the OpenFisca Web API with OpenFisca-France.
```

Comment:
This is actually every 12 hours, if I'm not mistaken 🙂
Let's avoid documentation rot altogether 😊

```suggestion
# Run the Ansible playbook that installs the OpenFisca Web API.
```